src/visuallyCompleteCalculator.ts

```diff
@@ -25,7 +25,20 @@ export type Metric = {
   };
 };
 
+export type Cancellation = {
+  start: number;
+  cancelled: number;
```

Comment:
`canceled` is actually the American spelling, although I suppose you're welcome to use British English 😀

Reply:
Nope, cancellation is the correct spelling regardless of country.